src/OutboundServer/api_template.yaml

```diff
@@ -0,0 +1,1330 @@
+openapi: 3.0.1
```

Comment:
now it looks much, much better, over 1200 lines cleaned :-D 
Obviously, endpoints are grouped around features, maybe we can split them into separate files and then merge them back in api_template (transfers, quotes, requestToPay, accounts...)? I believe it will bring much more clarity...